legal-api/report-templates/template-parts/amalgamation/approvalType.html

```diff
@@ -0,0 +1,30 @@
+<div class="no-page-break">
+    <div class="separator mt-4"></div>
+    <div class="section-title mt-4">
+      <span>Approval Type</span>
+    </div>
+    {% if amalgamationApplication.courtApproval is none %}
+      <div class="section-data mt-3">
+        N/A
+      </div>
+    {% else %}
+      {% if amalgamationApplication.courtApproval == 'courtOrder' %}
+        <div class="section-data mt-3">
+          The restoration was approved by a court order.
+        </div>
+        <div class="section-data mt-2">
+          Court order number: <span class="bold">{{amalgamationApplication.courtApproval.fileNumber}}</span>
+        </div>
+      {% elif amalgamationApplication.courtApproval == 'registrar' %}
+        <div class="section-data mt-3">
+          The restoration was approved by Registrar.
+        </div>
+        <div class="section-data mt-2">
+          BC Gazette published date: {{amalgamationApplication.courtApproval.noticeDate}}
+        </div>
+        <div class="section-data mt-2">
+          Notice of Application mailed date: {{amalgamationApplication.courtApproval.applicationDate}}
+        </div>
+      {% endif %}
```

Comment:
From looking at the designs, you can remove most of this code.  

If the amalgamation was approved by courtOrder, you just need to display "The amalgamation is approved by a court order" from what I can see in the designs.

![image](https://github.com/bcgov/lear/assets/6685223/cc28a01e-47c4-4fe7-adff-4dd27b51ac99)